src/gar.lua

```diff
@@ -96,6 +96,9 @@ GatewayRegistrySettings = GatewayRegistrySettings
 		},
 	}
 
+--- @type Timestamp|nil
+NextGatewaysPruneTimestamp = NextGatewaysPruneTimestamp or 0
```

Comment:
whats the main difference between 0 and nil here for the logic? 

```suggestion
--- @type Timestamp
NextGatewaysPruneTimestamp = NextGatewaysPruneTimestamp or 0
```

Reply:
It's an initial condition in this case. When we deploy updates to the source, globals will have previous values available to assign here. 0 indicates "definitely prune next time pruning happens". `nil` indicates that there's nothing to prune. 0 is more conservative when we're not sure what the deal should be.